docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,409 @@
+- The second filter drops all log records fulfilling the given rule. Here, typical Namespaces are dropped based on the `kubernetes` attribute.
+- A log record is modified by adding a new attribute. Here, a constant attribute is added to every log record for recording the actual cluster node name at the record for later filtering in the backend system. As value, a placeholder is used referring to a Kubernetes-specific environment variable.
+
+### Step 3: Add authentication details from Secrets
+
+Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the `LogPipeline` supports the reference of Secrets. The key-value entries of the Secrets can be mapped to environment variables of the Fluent Bit Pods, and with that, are available for usage in [placeholder expressions](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/variables).
+
+To leverage data provided by a Kubernetes Secrets in a custom output definition, use placeholder expressions for the data provided by the Secret. Then specify the actual mapping to the keys of the Secret in the `variables` section, like in the following example:
+
+```yaml
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
+spec:
+  filters:
+  ...
+  output:
+    custom: |
+      Name               http
+      Host               ${ENDPOINT} # Defined in Secret
+      HTTP_User          ${USER} # Defined in Secret
+      HTTP_Password      ${PASSWORD} # Defined in Secret
+      Tls                On
+  variables:
+    - name: ENDPOINT
+      valueFrom:
+        secretKeyRef:
+        - name: http-backend-credentials
+          namespace: default
+          key: ENDPOINT
+    ...
+```
+
+The related Secret must fulfill the referenced name and Namespace, and contain the mapped key as in the following example:
+
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: http-backend-credentials
+stringData:
+  HTTP_ENDPOINT: https://myhost/logs
+  HTTP_USER: myUser
+  HTTP_PASSWORD: XXX
+```
+
+### Step 4: Rotate the Secret
+
+A Secret being referenced with the `secretKeyRef` construct as used in the previous can be rotated manually or automatically. For automatic rotation, update the actual values of the Secret and keep the keys of the Secret stable.  
+Once an hour, the LogPipeline watches the referenced Secrets and detects changes to them. To enforce the detection, just annotate the LogPipeline; for example, with the following code:
+
+```yaml
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
+  annotations:
+    kyma-project.com/timeStamp: <current-time>
+```
+
+### Step 5: Add a parser
```

Comment:
The whole Step 5 renders as one paragraph, and I guess you want three separate ones, correct?
![image](https://user-images.githubusercontent.com/62639546/180416494-ca0ea441-07ff-4301-8420-aa2d07b15d3f.png)

Reply:
Indeed it was meant to be one paragraph